src/library/entities/Patron.java

```diff
@@ -1,118 +1,116 @@
-        return sb.toString();
-    }
-
-    
-    public Long GeT_ID() {
-        return PaTrOn_Id;
-    }
-
-    
-    public List<Loan> GeT_LoAnS() {
-        return new ArrayList<Loan>(cUrReNt_lOaNs.values());
-    }
-
-    
-    public int gEt_nUmBeR_Of_CuRrEnT_LoAnS() {
-        return cUrReNt_lOaNs.size();
-    }
-
-    
-    public double FiNeS_OwEd() {
-        return FiNeS_OwInG;
-    }
-
-    
-    public void TaKe_OuT_LoAn(Loan lOaN) {
-        if (!cUrReNt_lOaNs.containsKey(lOaN.getId()))
-            cUrReNt_lOaNs.put(lOaN.getId(), lOaN);
-        
-        else 
-            throw new RuntimeException("Duplicate loan added to member");
-                
-    }
-
-    public void dIsChArGeLoAn(Loan LoAn) {
-        if (cUrReNt_lOaNs.containsKey(LoAn.getId()))
-            cUrReNt_lOaNs.remove(LoAn.getId());
-        
-        else 
-            throw new RuntimeException("No such loan held by member");
-                
-    }
-    
-    public String GeT_LaSt_NaMe() {
-        return LaSt_NaMe;
-    }
-
-    
-    public String GeT_FiRsT_NaMe() {
-        return FiRsT_NaMe;
-    }
-
-
-    public void AdD_FiNe(double fine) {
-        FiNeS_OwInG += fine;
-    }
-    
-    public double PaY_FiNe(double AmOuNt) {
-        if (AmOuNt < 0) 
-            throw new RuntimeException("Member.payFine: amount must be positive");
-        
-        double cHaNgE = 0;
-        if (AmOuNt > FiNeS_OwInG) {
-            cHaNgE = AmOuNt - FiNeS_OwInG;
-            FiNeS_OwInG = 0;
-        }
-        else 
-            FiNeS_OwInG -= AmOuNt;
-        
-        return cHaNgE;
-    }
-
-
-
-}
+package library.entities;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@SuppressWarnings("serial")
+public class Patron implements Serializable {
+
+    private long patronId;
+    private String firstName;
+    private String lastName;
+    private String emailAddress;
+    private long phoneNumber;
+    private double finesOwing;
+    private Map<Long, Loan> currentLoans;
+
+    
+    public Patron(String firstName, String lastName, String emailAddress, long phoneNumber, long memberId) {
+        this.firstName = firstName;
+        this.lastName = lastName;
+        this.emailAddress = emailAddress;
+        this.phoneNumber = phoneNumber;
+        this.patronId = memberId;
+        this.currentLoans = new HashMap<>();
+    }
+
+    
+    public String toString() {
+        StringBuilder sb = new StringBuilder();
+        sb.append("Patron:  ").append(patronId).append("\n")
+          .append("  Name:  ").append(firstName).append(" ").append(lastName).append("\n")
+          .append("  Email: ").append(emailAddress).append("\n")
+          .append("  Phone: ").append(phoneNumber)
+          .append("\n")
+          .append(String.format("  Fines Owed :  $%.2f", finesOwing))
+          .append("\n");
+        
+        for (Loan LoAn : currentLoans.values()) {
```

Comment:
PR#36 R1-D2-CSG#2: All variable names are to start with a lowercase letter and to be in camelBack
hint:LoAn to be loan.

Reply:
Fixed here with this commit[https://github.com/elliotmitchum/ITC205/pull/36/commits/a3ed401a1c8b98aa6e2231bb45909088ecaa6e7e](url)